src/atomate2/vasp/sets/mp.py

```diff
@@ -44,6 +43,45 @@ class MPGGAStaticSetGenerator(StaticSetGenerator):
     config_dict: dict = field(default_factory=lambda: _BASE_MP_GGA_RELAX_SET)
 
 
+@dataclass
+class MPMetaGGAStaticSetGenerator(StaticSetGenerator):
+    """Class to generate MP-compatible VASP GGA static input sets."""
+
+    config_dict: dict = field(default_factory=lambda: _BASE_MP_R2SCAN_RELAX_SET)
+    auto_ismear: bool = False
+
+    def get_incar_updates(
+        self,
+        structure: Structure,
+        prev_incar: dict = None,
+        bandgap: float = None,
+        vasprun: Vasprun = None,
+        outcar: Outcar = None,
+    ) -> dict:
+        """
+        Get updates to the INCAR for this calculation type.
+
+        Parameters
+        ----------
+        structure
+            A structure.
+        prev_incar
+            An incar from a previous calculation.
+        bandgap
+            The band gap.
+        vasprun
+            A vasprun from a previous calculation.
+        outcar
+            An outcar from a previous calculation.
+
+        Returns
+        -------
+        dict
+            A dictionary of updates to apply.
+        """
+        return {"ALGO": "FAST"}
```

Comment:
@esoteric-ephemera @arosen93 Do you remember any reason why we don't set this in the `_BASE_MP_R2SCAN_RELAX_SET` yaml config?

Reply:
Presumably because we want to use ALGO = "All" with R2SCAN and only use ALGO = "Fast" with GGA. So, we have ALGO = "All" in the set but override it just for the GGA step. Note: This is potentially something where inheritance can be tricky again...

Reply:
But this isn't GGA, this is the SCAN calculation.

Reply:
I'll move `ALGO=FAST` into `src/atomate2/vasp/sets/BaseMPR2SCANRelaxSet.yaml` then

Reply:
Actually, I guess we wanted `ALGO=ALL` in `MPMetaGGARelaxSetGenerator` and only set `ALGO=FAST` in `MPMetaGGAStaticSetGenerator`?